files/setup-05-event-processor.sh

```diff
@@ -22,7 +22,52 @@ ESCAPED_VCENTER_USERNAME=$(echo -n ${VCENTER_USERNAME} | python -c 'import sys,j
 ESCAPED_VCENTER_PASSWORD=$(echo -n ${VCENTER_PASSWORD} | python -c 'import sys,json;data=sys.stdin.read(); print json.dumps(data)[1:-1]')
 ESCAPED_ROOT_PASSWORD=$(echo -n ${ROOT_PASSWORD} | python -c 'import sys,json;data=sys.stdin.read(); print json.dumps(data)[1:-1]')
 
-if [ "${EVENT_PROCESSOR_TYPE}" == "AWS EventBridge" ]; then
+if [ "${EVENT_PROCESSOR_TYPE}" == "Knative" ]; then
+    if [ -z ${KNATIVE_HOST} ]; then
+      echo "Knative Host was not provided, exiting ..."
+      exit 1
+    fi
+
+    echo -e "\e[92mSetting up Knative Processor ..." > /dev/console
+
+    cat > ${EVENT_ROUTER_CONFIG} << __KNATIVE_PROCESSOR__
+apiVersion: event-router.vmware.com/v1alpha1
+eventProcessor:
+  name: veba-knative
+  type: knative
+  knative:
+    insecureSSL: ${KNATIVE_DISABLE_TLS}
```

Comment:
Is this coming in as boolean or converted from string? If the latter, are we at risk of dealing with empty string `""`?

Reply:
Boolean, the OVF will always be set to either True/False, so we're fine here